plugins/chat/app/services/chat/publisher.rb

```diff
@@ -10,8 +10,27 @@ def self.root_message_bus_channel(chat_channel_id)
       "/chat/#{chat_channel_id}"
     end
 
+    def self.thread_message_bus_channel(chat_channel_id, thread_id)
+      "#{root_message_bus_channel(chat_channel_id)}/thread/#{thread_id}"
+    end
+
+    def self.calculate_publish_targets(chat_channel, chat_message)
+      targets =
+        if chat_message.thread_om?
+          [
+            root_message_bus_channel(chat_channel.id),
+            thread_message_bus_channel(chat_channel.id, chat_message.thread_id),
+          ]
+        elsif chat_message.thread_reply?
+          [thread_message_bus_channel(chat_channel.id, chat_message.thread_id)]
+        else
+          [root_message_bus_channel(chat_channel.id)]
+        end
+      targets
+    end
```

Comment:
That's nitpick but I really prefer if we use channel over `chat_channel` and message over `chat_message`, this verbosity is not needed in this context